.github/workflows/ci.yml

```diff
@@ -157,13 +154,8 @@ jobs:
         # it can be used as a docker tag
         run: echo "::set-output name=branch::$(echo ${GITHUB_REF#refs/heads/} | tr / -)"
 
-      - name: Set up Docker
-        run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker
-          sudo /Applications/Docker.app/Contents/MacOS/Docker --unattended --install-privileged-components
-          open -a /Applications/Docker.app --args --unattended --accept-license
-          echo "We are waiting for Docker to be up and running. It can take over 2 minutes..."
-          while ! /Applications/Docker.app/Contents/Resources/bin/docker info &>/dev/null; do sleep 1; done
+      - name: enable arm64 building
+        run: docker run --rm --privileged tonistiigi/binfmt --install arm64
```

Comment:
I'm glad you figured this out!

@asottile-sentry I never quite understood the docs of that project. What does it do?

The [doc](https://github.com/tonistiigi/binfmt) for the curious.

Reply:
it does some... kinda sketch things to the docker daemon to allow it to use qemu to run alternative architecture images

iirc Docker.app ships with these modifications already in place

Reply:
Acrobatic automation! You never know what kind of pirouettes you need to perform in order to get to the finish line 😆 

Thanks Anthony! I feel better with this.